### DISCUSSION TITLE
memos_prod.db database file

### DISCUSSION THREAD

**Discussion by VlaK0r** [usememos/memos#1948] [Q&A] [answered]
Hello!
Thank you for the wonderful product!
What information is stored in the memos_prod.db file? Only the text or the entire contents of the notes?

**Comment by athurg** [comment]
This database file store all data of Memos by default.

But if you change the system setting to save resource other than `database` (currently only `database`/`local`/`s3` are supported), the resource blob will not be included in the database file.

**Reply by VlaK0r** [reply]
I didn't quite get it. 
In this case, where is saved?
![image](https://github.com/usememos/memos/assets/77666575/658de90b-5598-4cb0-91da-c149bdba596a)

**Reply by athurg** [reply]
Everything store in the database as you config.

**Reply by VlaK0r** [reply]
In memos_prod.db ?

**Reply by athurg** [reply, marked as answer]
Yes, you're completely right!

**Reply by VlaK0r** [reply]
And if the locale?
Text in the memos_prod.db, and pictures in the folder?

**Reply by athurg** [reply]
If you config `local` (not locale) in the *Current object storage*, the resource (include pictures) store in the disk folder, but the meta info of the resource, such as filename or size will still keep in the database file.

**Reply by VlaK0r** [reply]
Text in database file?

**Reply by athurg** [reply]
Yes, text always in database file

**Reply by VlaK0r** [reply]
Thank you very much!

**Comment by dwong33** [comment]
I want to ask the structure of db file, what is the purpose of **memo_organizer** ? There are two records of memos and these two are always shown at the bottom of the home page.

![image](https://github.com/usememos/memos/assets/12155059/28f54d2a-acc5-4dfc-af7d-c72371eb7f83)
